python/tirex_tracker/__init__.py

```diff
@@ -848,6 +860,18 @@ def _export_ir_metadata(self, result: "Pointer[_Result]") -> None:
         ir_metadata = _deep_merge(ir_metadata, tmp_ir_metadata)
         ir_metadata = _recursive_undefaultdict(ir_metadata)
 
+        global REGISTERED_METADATA
+        if REGISTERED_METADATA:
+            ir_metadata.update(REGISTERED_METADATA)
+
+        for r_resolve_dir, r_file, subdir in REGISTERED_FILES:
+            if subdir is None:
+                target_file = self._export_file_path.parent / r_file
+            else:
+                target_file = self._export_file_path.parent / subdir / r_file
+            target_file.parent.mkdir(exist_ok=True, parents=True)
+            shutil.copy(r_resolve_dir/r_file, target_file)
```

Comment:
With qualified imports, this will be just `copy(r_resolve_dir / r_file, target_file)`